rust/cubestore/cubestore/src/config/mod.rs

```diff
@@ -1184,7 +1188,7 @@ impl Config {
             },
             meta_store: self.injector.get_service_typed().await,
             cluster: self.injector.get_service_typed().await,
-            remote_fs: self.injector.get_service_typed().await,
+            //remote_fs: self.injector.get_service_typed().await,
```

Comment:
Drive by comment: As a general rule, if code is no longer needed, let's just delete it.